---a/net/minecraft/network/play/server/S1EPacketRemoveEntityEffect.java
+++b/net/minecraft/network/play/server/S1EPacketRemoveEntityEffect.java
@@ -1,48 +1,44 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.potion.PotionEffect;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S1EPacketRemoveEntityEffect implements Packet
-{
+
+public class S1EPacketRemoveEntityEffect implements Packet {
+
     private int field_149079_a;
     private int field_149078_b;
     private static final String __OBFID = "CL_00001321";
 
-    public S1EPacketRemoveEntityEffect() {}
+    public S1EPacketRemoveEntityEffect() {
+    }
 
-    public S1EPacketRemoveEntityEffect(int p_i45212_1_, PotionEffect p_i45212_2_)
-    {
+    public S1EPacketRemoveEntityEffect(int p_i45212_1_, PotionEffect p_i45212_2_) {
         this.field_149079_a = p_i45212_1_;
         this.field_149078_b = p_i45212_2_.getPotionID();
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.field_149079_a = buf.readVarIntFromBuffer();
         this.field_149078_b = buf.readUnsignedByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeVarIntToBuffer(this.field_149079_a);
         buf.writeByte(this.field_149078_b);
     }
 
-    public void processPacket(INetHandlerPlayClient handler)
-    {
+    public void processPacket(INetHandlerPlayClient handler) {
         handler.handleRemoveEntityEffect(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public int func_149076_c()
-    {
-        return this.field_149079_a;
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayClient) handler);
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayClient)handler);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public int func_149075_d()
-    {
-        return this.field_149078_b;
-    }
-}
+}
